cmd/run/run.go

```diff
@@ -415,20 +412,7 @@ func RunServer(ctx context.Context, config *Config) error {
 		experimentals = append(experimentals, server.ExperimentalFeatureFlag(feature))
 	}
 
-	meter := metric.NewNoopMeter()
 	var err error
```

Comment:
Can you move this to closer to where it is used?

Reply:
Yep, done.